content/components/overlay.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Overlay
+status: Alpha
+---
+
+import {Grid, Flex, Box, Button, ButtonOutline, Heading, Label, LabelGroup, Link} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    An overlay is a flexible floating surface, used to display transient content such as menus, selection options, dialogs, and more. Overlays use shadows to express elevation, and may include a backdrop that dims the rest of the screen for focus. 
+</Box>
+
+<LabelGroup display="block" mb={4}>
+    <Label as="a" href="https://primer.style/components/ActionList" variant="xl" color="text.success" outline style="text-decoration: none; line-height: 20px;">
+        <img width="20" height="20" alt=" " src="https://user-images.githubusercontent.com/293280/123878374-ce9d4d00-d8f3-11eb-8adf-1a160292ff53.png" style="vertical-align: middle; margin-right: 4px;" />
+        React
+    </Label>
+    <Label as="a" href="https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=1927%3A0" variant="xl" outline style="text-decoration: none; line-height: 20px;">
+        <img width="20" height="20" alt=" " src="https://user-images.githubusercontent.com/293280/123878720-80d51480-d8f4-11eb-9b10-d02a1cb606f8.png" style="vertical-align: middle; margin-right: 4px;" />
+        Figma
+    </Label>
+</LabelGroup>
+
+<!--
+<Label as="a" href="/components/ActionList" variant="xl" outline style="text-decoration: none">ViewComponent</Label>
+
+-->
+
+<img src="https://user-images.githubusercontent.com/293280/125706076-b16bb3d4-800f-4310-a082-f576e215e6c8.png" alt="Overlay examples" />
+
+## Overview
+
+Overlays can streamline the interface by allowing extra options to be displayed as the person needs them. Overlays present a new context to the user, and can facilitate the completion of individual tasks.
+
+**Keep overlays simple**. Overlays are meant to be interacted quickly, so they're better suited for focused experiences. Avoid creating a whole page inside an overlay. Prefer single-column layouts that convey a clear goal the moment it's opened.
+
+**Overlays are often used for data manipulation**. Overlays can appear on top of the screen to let the user select and edit content in a way that was previously not available. Ensure its interactions are quick and effective.
+
+**Overlays can be fully interacted with a keyboard**. Keyboard navigation should be limited to the elements of the active overlay. If the overlay contains a list, let users navigate using keyboard arrows. Pressing Escape should dismiss the overlay. When an overlay is dismissed, focus generally returns to the element that opened it.
+
+**Overlay contents should work well on any device**. Overlays may transform into bottom sheets or full-screen experiences on mobile devices. Make sure the experience takes into account devices of all sizes and input methods.
+
+
+## Anatomy
+
+An overlay may wrap its content directly, or include a header to provide content, and a footer to provide actions.
```

Comment:
```suggestion
An overlay may include a header to provide content, and a footer to provide actions.
```
I removed the first part of the sentence because it was making it hard to parse. If that content is important, I recommend writing it in a simpler way.